Sources/AppcuesKit/Presentation/Public/Plugins/AppcuesExperiencePluginConfiguration.swift

```diff
@@ -28,9 +28,13 @@ public class AppcuesExperiencePluginConfiguration: NSObject {
     /// The context where the plugin was defined.
     public let level: Level
 
-    init(_ decoder: PluginDecoder, level: Level) {
+    /// The instance of the Appcues SDK where the plugin is being applied.
+    public weak var appcues: Appcues?
```

Comment:
discussion ongoing, but now that the Appcues instance is here, we do not technically need to also include in calls to AppcuesExperienceAction `execute(inContext, completion)`

Reply:
decision is to remove the appcues instance from the execute function signature, another change coming on top of this for that update.